Documentation/teaching/labs/block_device_drivers.rst

```diff
@@ -1139,14 +1140,7 @@ the pages of each :c:type:`struct bio` structure and access its associated
 buffers. For the actual transfer, call the :c:func:`my_block_transfer` function
 implemented in the previous exercise.
 
-For testing, use the :file:`ram-disk-test.c` test file. Compile it on
-the host, using the command:
-
-.. code-block:: shell
-
-    make -f Makefile.test
-
-and then run it in the QEMU virtual machine using the following command:
+For testing, use the :file:`ram-disk-test.c` test file:
```

Comment:
Why did you remove the instructions to compile the test file? Are they presented earlier?

Reply:
Yes, they are described for exercise 3.